modules/imgimport/CMakeLists.txt

```diff
@@ -1,2 +1,4 @@
 include_directories(include ../core/include)
+include_directories(include ../core/src)
+include_directories(include ./include)
```

Comment:
You only need to call include_directories once. It takes multiple arguments.
You also shouldn't include source directories.